content/articles/mastering-yara-for-malware-analysis-the-right-way/index.md

```diff
@@ -0,0 +1,160 @@
+According to security researchers, over 450,000 new malicious programs are discovered in a day. These figures continue rising each year. Currently, about 1343.64 malicious programs have been reported(April 2022).
+*(Source: [av-test, 2022](https://www.av-test.org/en/statistics/malware/))*
+
+Malicious programs, commonly known as malware, can be defined as software developed intentionally to harm computer systems or alter their normal mode of operation. We can spread malware through various methods, such as downloads from pirated sites, storage media, i.e. USB drives, and emails with malicious links and attachments that appear to be from legitimate sources, among others.
+
+Computer systems infected with malware often display some signs. You may notice that your computer is getting slower, settings on some of your applications have been altered, some files are missing, or existence of new unknown files among other signs.
+
+In this article, we look at YARA, a malware analysis tool used to detect patterns of malware characteristics in files, using a rule-based approach. We will focus on What YARA is, How to install it in windows and Linux environments, and finally handle its syntax.
+
```

Comment:
Add a "Table of contents" and "Prerequisites" section here
Check out this [guide](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

Reply:
@Jethro-magaji I have updated the structure of the article.